src/main/resources/org/broadinstitute/barclay/helpTemplates/generic.ftl

```diff
@@ -4,7 +4,7 @@
 
 <div class='row-fluid' id="top">
 
-	<#include "common.html"/>
+	<#include "common.ftl"/>
```

Comment:
I think that you are referring to the top php line in the generic template, because this is an `#include` for FreeMaker and the `common.ftl` a set of macros used in both the generic and index templates.